inc/namespace.php

```diff
@@ -19,15 +20,56 @@ function bootstrap() {
 function on_plugins_loaded() {
 	$config = get_config()['modules']['dev-tools'];
 
-	if ( $config['query-monitor'] ) {
-		// In Cloud environments, disable the Query Monitor error handler
-		// as this will override our own error handler in Cloud.
-		if ( in_array( get_environment_architecture(), [ 'ec2', 'ecs' ], true ) ) {
-			define( 'QM_DISABLE_ERROR_HANDLER', true );
-		}
+	if ( ! $config['query-monitor'] ) {
+		return;
+	}
+
+	// In Cloud environments, disable the Query Monitor error handler
+	// as this will override our own error handler in Cloud.
+	if ( in_array( get_environment_architecture(), [ 'ec2', 'ecs' ], true ) ) {
+		define( 'QM_DISABLE_ERROR_HANDLER', true );
+	}
+
+	// Hide the db.php dropin installation warning and prompt.
+	add_filter( 'qm/show_extended_query_prompt', '__return_false' );
+	require_once ROOT_DIR . '/vendor/johnbillion/query-monitor/query-monitor.php';
+
+	// Add Altis Config info to the Query Monitor.
+	// Register collector.
+	add_filter( 'qm/collectors', __NAMESPACE__ . '\\register_qm_collector_altis_config' );
```

Comment:
It's a minor thing and totally optional but I feel like these function names could made into more readable english sentences eg. `register_altis_config_qm_collector` and `register_altis_config_qm_output_html`.

This reads more naturally and will group them in autocomplete dropdowns too.